src/ucp/tag/rndv.c

```diff
@@ -1317,8 +1318,14 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rtr_handler,
                       ucp_ep_peer_name(ep), ucs_status_string(status));
         }
 
+        is_pipeline_rndv = ((!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type) ||
+                             (sreq->send.length != rndv_rtr_hdr->size)) &&
```

Comment:
minor: can remove braces around 
```
!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type
```
Also second condition is unaligned - 1 extra whitespace at the begining of the statement.

Reply:
this is copy paste from prev code and braces are ok (the condition logic itself seems "hairy" but we can discuss it later)